File: Upskill/UpSkillWebApi/UpSkillWebApi/Models/Review.cs
using System.ComponentModel.DataAnnotations;

namespace UpSkillWebApi.Models
{
    public class Review
    {
        [Key]
        public int ReviewId { get; set; }   
        public int CourseId { get; set; }
        public string Author { get; set; }
        public DateTime Date { get; set; }
        public int Rating { get; set; }
        public string Description { get; set; }
    }
}

File: Angular-Components/UpSkillUi/src/app/models/Review.ts
import { Component } from "@angular/core";

export class Review{
    reviewId:number;
    courseId:number;
    author:string;
    date:Date;
    rating:number;
    description:string;
    constructor(reviewId:number,courseId:number,author:string,date:Date,rating:number,description:string){
       this.reviewId=reviewId;
       this.courseId=courseId;
       this.author=author;
       this.date=date;
       this.rating=rating;
       this.description=description;

    }
}